app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt

```diff
@@ -0,0 +1,90 @@
+package com.scrumptious.algorithmvisualizer
+
+fun bubbleSortSequenceGenerator(inputArr: Array<Int>): Array<Pair<Array<Int>, Array<Int>>> {
```

Comment:
Consider making this a class at some point

Reply:
I have created the class and placed the function within it.